### PR TITLE
Added new widgets

### PR DESCRIPTION
**Pull request by shaniacht1**

### CODE REVIEW

File: Widgets/widget-SLAbyType.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "period": {
+      "byTo": "",
+      "byFrom": "days",
+      "toValue": null,
+      "fromValue": 7,
+      "field": ""
+    }
+  },
+  "params": {
+    "groupBy": ["sla","type"]
```
Comment (Bars92) [inline Widgets/widget-SLAbyType.json]: please tests this. I think it should be `dbotDueDate` rather then `sla`

File: Widgets/widget-MttrByType.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "period":{
+      "byTo":"",
+      "byFrom":"days",
+      "toValue":null,
+      "fromValue":7,
+      "field":""
+    }
+  },
+  "params":{
+     "keys":["avg|openDuration", "count|1"],
```
Comment (Bars92) [inline Widgets/widget-MttrByType.json]: i think we dont need the count here. We need `avg|openDuration / (3600*24)` to get avg open duration in days (defiantly not in milliseconds)

Comment (Bars92) [reply Widgets/widget-MttrByType.json]: Also make sure for other widgets. Please test them

**Comment by meirwah** [comment]
@Bars92 plz review/approve this

**Pull request merged**